packages/block-editor/src/components/block-list/use-block-props/use-focus-first-element.js

```diff
@@ -96,6 +106,25 @@ export function useFocusFirstElement( clientId ) {
 			return;
 		}
 
+		// Check to see if element is focussable before a generic caret insert.
+		if ( ! target.getAttribute( 'contenteditable' ) ) {
+			const focusElement = focus.tabbable.findNext( target );
+			// Ensure is not block inserter trigger, don't want to focus that in the event of the group block which doesn't contain any other focussable elements.
+			const skipBlockInserterTrigger = target.classList.contains(
```

Comment:
This has been something of a terrible pain. For some reason, hasAttribute/getAttribute throw errors on tests so I gave up and just used classList. I attempted to get it working with aria-label but for whatever reason, this is way easier in tests than it is in practice.